### DISCUSSION TITLE
Version 1.10.1rc1

### DISCUSSION THREAD

**Discussion by jpivarski** [scikit-hep/awkward#1711] [New releases]
## New features

_(none)_

## Bug-fixes and performance

* fix: support non-hashable parameter values in broadcasting by @agoose77 in https://github.com/scikit-hep/awkward/pull/1708

## Other

* chore: update pre-commit hooks by @pre-commit-ci in https://github.com/scikit-hep/awkward/pull/1706

**Full Changelog**: https://github.com/scikit-hep/awkward/compare/v1.10.0...v1.10.1rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/scikit-hep/awkward/releases/tag/v1.10.1rc1'>Version 1.10.1rc1</a>.</em>

**Comment by philippemiron** [comment]
Just to confirm that this fixes the issues I had with 1.10.0, cheers.
